docs/docs/understanding/workflows/concurrent_execution.md

```diff
@@ -24,6 +24,29 @@ class ParallelFlow(Workflow):
 
 In this example, our `start` step emits 3 `StepTwoEvent`s. The `step_two` step is decorated with `num_workers=4`, which tells the workflow to run up to 4 instances of this step concurrently (this is the default).
 
+## The `dispatch` method
+
+To simplify the multiple `send_event` calls that we used in the previous examples, we also provide a `dispatch` method within the `Context` itself:
+
+```python
+class ParallelFlow(Workflow):
+    @step
+    async def start(self, ctx: Context, ev: StartEvent) -> StepTwoEvent | None:
+        events = [
+            StepTwoEvent(query="Query 1"),
+            StepTwoEvent(query="Query 2")
+            StepTwoEvent(query="Query 3")
+        ]
+        await ctx.dispatch(events)
+
+    @step(num_workers=4)
+    async def step_two(self, ctx: Context, ev: StepTwoEvent) -> StopEvent:
```

Comment:
won't this stop the workflow after the first event since this returns a `StopEvent`? Might be better to leave it open ended/incomplete rather than returning a stop event